site/content/integrate/admin-guide/admin-slash-commands/_index.md

```diff
@@ -60,6 +60,8 @@ You can follow these general guidelines to set up a custom Mattermost slash comm
 4. Set the **Request URL** and **Request Method**. The request URL is the endpoint that Mattermost hits to reach your application, and the request method is either POST or GET and specifies the type of request sent to the request URL.
 5. (Optional) Set the response username and icon the command will post messages as in Mattermost. If not set, the command will use your username and profile picture.
 
+**Important:** Custom command URLs do not support basic authentication as the slash commands provide their own authentication level.
```

Comment:
What does this mean @justinegeffen @mickmister

Reply:
@aaronrothschild I honestly don't know. We use a secret when we send the request but that's it. Looks like the original PR https://github.com/mattermost/docs/pull/3190 had no dev reviewers so the claim/wording was not vetted for correctness. @justinegeffen Do you know why this specific sentence was taken from the other PR, but none of the other sentences from the PR?

Reply:
@justinegeffen Maybe we should close this PR?